api_spec.yaml

```diff
@@ -322,12 +323,18 @@ paths:
                       $ref: '#/components/schemas/createJWTPayload'
                   required:
                     - authorizationCode
+                    - clientId
                 - type: object
```

Comment:
in the response, you don't need to give userID, sessionHandle and scopes.

Reply:
Need to confirm this @porcellus . 
As per our last discussion, 
- userId is added in response if the grant_type is implicit, (i think if userId can be included in claims, so this field can be excluded)
- sessionHandle is added for rest of the grant types , excluding implicit .
- some oauth providers do add scopes in response and its a good to have feature.

Also , i think we have missed out `token_type` field.

Reply:
> userId is added in response if the grant_type is implicit, (i think , userId can be included in claims, so this field can be excluded)

We don't support implicit grant - so i have no idea why this is coming up.

Reply:
Changes made